src/main/kotlin/no/nav/familie/ef/sak/vilkår/dto/VilkårDto.kt

```diff
@@ -9,6 +11,7 @@ data class VilkårDto(
 )
 
 data class VilkårGrunnlagDto(
+    val registergrunnlag: VilkårRegistergrunnlagDto,
```

Comment:
Kan vi kalle det noe annet enn registergrunnlag? For medlemskap, sivilstand, bosituasjon, barnmedsamvær osv her jo også alle registergrunnlag, men ligger innunder et avgrenset "domene" . 

Er navn og perosnident en del av `medlemskap.registergrunnlag` ? Og er bostedsadresse en de lav adresseopplysninger? Der ligger det jo adresse allerede i dag...

Reply:
Hvis det ikke passer inn under andre dto-er - kan det heller hete `personalia` eller noe slikt?

Reply:
Enig med registergrunnlag :) 
Jeg la til adressen først og då under adresseopplysninger, men pga div logikk koblet til undefined eller ikke på den delen så tenkte jeg det var enklere å legge det et annet sted.
Tenker navn og personident passer fint under noe som heter personalia, og tenker adressen også kan ligge der?

Reply:
Da skjønner jeg, kall det gjerne personalia og ta med adressen inn der :)